packages/lib-classifier/src/plugins/tasks/dropdown-simple/components/SimpleDropdownTask.spec.js

```diff
@@ -1,108 +1,83 @@
+import { render, screen } from '@testing-library/react'
+import { within } from '@testing-library/dom'
+import userEvent from '@testing-library/user-event'
+import { composeStory } from '@storybook/react'
+import Meta, { ShortOptions, MediumOptions, LongOptions } from './SimpleDropdownTask.stories.js'
+import {
+  SimpleDropdownTaskMockShort,
+  SimpleDropdownTaskMockMedium,
+  SimpleDropdownTaskMockLong
+} from './SimpleDropdownTask.mock'
+
+describe('Tasks / Simple Dropdown', function () {
+  const user = userEvent.setup()
+
+  describe('Short list of options', function () {
+    const ShortOptionsStory = composeStory(ShortOptions, Meta)
+
+    beforeEach(function () {
+      render(<ShortOptionsStory />)
     })
 
-    it('should render without crashing', function () {
-      expect(wrapper).to.be.ok()
+    it('should render without any option selected', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      expect(inputEl.value).to.equal('')
     })
 
-    it('should have a question/some instructions', function () {
-      expect(wrapper.contains(task.instruction)).to.be.true()
+    it('should render the full list of options', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      await user.click(inputEl)
+
+      const options = screen.getAllByRole('option')
+
+      expect(options.length).to.equal(SimpleDropdownTaskMockShort.options.length)
+      expect(within(options[0]).getByText(SimpleDropdownTaskMockShort.options[0])).to.exist()
     })
+  })
 
-    it('should render a single Grommet Select element', function () {
-      expect(wrapper.find('Select')).to.have.lengthOf(1)
+  describe('Medium list of options', function () {
+    beforeEach(function () {
+      const MediumOptionsStory = composeStory(MediumOptions, Meta)
+      render(<MediumOptionsStory />)
     })
-    
-    it('should have no initial value', function () {
-      const grommetSelect = wrapper.find('Select')
-      expect(grommetSelect.props()['value']).to.equal(undefined)
+
+    it('should render without any option selected', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      expect(inputEl.value).to.equal('')
     })
-    
-    it('should render the correct number of options', function () {
-      const grommetSelect = wrapper.find('Select')
-      const renderedOptions = grommetSelect.props()['options'] || []
-      expect(renderedOptions).to.have.length(6)
+
+    it('should render the full list of options', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      await user.click(inputEl)
+
+      const options = screen.getAllByRole('option')
+
+      // Grommet Select only renders up to 20 options in the test environment, SHOULD be 40
```

Comment:
Because Grommet does not let you truly test the MediumOptionsStory, I recommend deleting its unit tests. They're just a replica of ShortOptionsStory unit tests.